coalib/parsing/ConfParser.py

```diff
@@ -113,7 +113,7 @@ def __parse_lines(self, lines, origin):
                         Setting(key,
                                 value,
                                 origin,
-                                # Ignore PEP8Bear, it fails to format that
+                                # Ignore PEP8Bear and PycodestyleBear*, it fails
```

Comment:
If i let the explanation as it was, it was exceeding the line width of 80 characters and the ignore comment only affects the line after it. But I can fix it by writing "# Start ignoring... # Stop ignoring".